Src/Card/Card.cpp

```diff
@@ -1 +1,136 @@
-#include "Card.h"
\ No newline at end of file
+#include "Card.h"
+//Constructor
+Card::Card(){
+
+}
+Card::Card(const string &IdAccount, const string &IdHolder, const string &pin, const int &balance)
```

Comment:
Nên validate trước khi gán. Ví dụ hàm constructor của Client
```suggestion
Client::Client(const string &ID, const string &name, const string &gender, const string &CCCD, const Date &birth){
    this -> ID = (Client::isValidID(ID)) ? ID : "";
    this -> name = (Client::isValidName(name)) ? name : "";
    this -> gender = (Client::isValidGender(gender)) ? Client::formatGender(gender) : "Other";
    this -> CCCD = CCCD;
    this -> birth = (Date(birth).isValidDate()) ? birth : Date();
    this -> createdAt = Date::getCurrentDate();
    this -> updatedAt = Date();
}
```